app/models/interviewee.rb

```diff
@@ -0,0 +1,10 @@
+class Interviewee < ApplicationRecord
+    has_one_attached :resume
```

Comment:
R we storing all files of any type? Don't u think we should strict this to some type?